src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector64.cs

```diff
@@ -2271,6 +2271,25 @@ public static Vector64<sbyte> Shuffle(Vector64<sbyte> vector, Vector64<sbyte> in
             return result;
         }
 
+        /// <summary>Creates a new vector by selecting values from an input vector using a set of indices.
+        /// Behavior is platform-dependent for out-of-range indices.</summary>
+        /// <param name="vector">The input vector from which values are selected.</param>
+        /// <param name="indices">The per-element indices used to select a value from <paramref name="vector" />.</param>
+        /// <returns>A new vector containing the values from <paramref name="vector" /> selected by the given <paramref name="indices" />.</returns>
+        /// <remarks>Unlike Shuffle, this method delegates to the underlying hardware intrinsic without ensuring that <paramref name="indices"/> are normalized to [0, 7].
+        /// On hardware with <see cref="AdvSimd"/> support, this method behaves the same as Shuffle.</remarks>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        [CompExactlyDependsOn(typeof(AdvSimd))]
+        public static Vector64<byte> ShuffleUnsafe(Vector64<byte> vector, Vector64<byte> indices)
+        {
+            if (AdvSimd.IsSupported)
+            {
+                return AdvSimd.VectorTableLookup(Vector128.Create(vector, Vector64<byte>.Zero), indices);
+            }
```

Comment:
Does the regular `Shuffle` struggle from the same codegen issues as `Vector128` and friends? Or can we drop this special case here and just call into `Shuffle`?
Presumably it should also always compile down to the same instruction on Arm.

Reply:
`Vector128` does this, so I assumed there was a reason to? But I'm not sure, couldn't get godbolt to give me arm64 assembly.

Reply:
Yeah, they both seem to give up for whatever reason:
https://github.com/dotnet/runtime/blob/be0e14cdd60d6e11e4ee13d96d748bfc0efffbc1/src/coreclr/jit/hwintrinsicarm64.cpp#L1630-L1652